packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -120,19 +115,12 @@
     .label:hover {
       .label-text::before {
         border-color: var(--terra-form-checkbox-hover-border-color, none);
-
-        [dir=rtl] & {
```

Comment:
What's the logic around removing the dir selectors? I thought these were needed to fix some issues where initial value is non-bidirectional, e.g. `none` and then the themed value includes styles which would be bidirectional?